ClientIdentifier.ts

```diff
@@ -0,0 +1,16 @@
+import { Identifier } from "authly"
+
+export abstract class ClientIdentifier {
+	private constructor() {}
+	static set id(id: string) {
```

Comment:
Call the property `value` instead or call the class `Client` and the property `identifier`.

Reply:
We need to decide if we call it `id` or `identifier`. It should be one or the other, never both. @fredrikbryntesson